jib-core/src/test/java/com/google/cloud/tools/jib/frontend/JavaDockerContextGeneratorTest.java

```diff
@@ -73,7 +73,7 @@ private static void assertSameFiles(Path directory1, Path directory2) throws IOE
         .map(
             sourceFile ->
                 new LayerEntry(
-                    sourceFile, extractionRoot.resolve(directory.relativize(sourceFile))))
+                    sourceFile, extractionRoot.resolve(directory.relativize(sourceFile)), null))
```

Comment:
I wonder if this Dockerfile generator should be writing out `RUN chmod xxx` for the various files…

Reply:
Hmm... that's a good point. But probably distroless doesn't have `chmod`, so it may not work in the default case. Not sure what we can actually do.

Reply:
I think we should actually give a warning that file permissions are not supported  when the user tries to export a docker context with file permissions configured.